backend/lib/til_web/guardian.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Til.Guardian do
+  use Guardian, otp_app: :til
+  alias Til.Accounts
+
+  def subject_for_token(email, _claims) do
+    sub = to_string(email)
```

Comment:
I wonder if we should have email as a `sub` claim? what do you think of using an ID?

Reply:
we could even generate UUID when creating users and expose only UUID rather than database id

Reply:
Nice idea, implementing uuid generation.